app/views/passengers/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<h1>Passenger Details</h1>
+
+<ul class="passenger-details">
+  <li><strong>Passenger ID: </strong>
+    <%= @passenger.id %>
+  </li>
+  <li><strong>Name: </strong>
+    <%= @passenger.name %>
+  </li>
+  <li><strong>Phone Number: </strong>
+    <%= @passenger.phone_num %>
+  </li>
+  <li><strong>Net Spending: </strong>
+    $ <%= @passenger.net_expenditures %>
+  </li>
+  <li>
+    <%= link_to "Edit", edit_passenger_path(@passenger.id) %>
+  </li>
+  <li>
+    <%= link_to "Delete", passenger_path(@passenger.id), method: :delete, data: { confirm: "Do you want to delete #{@passenger.name}?" } %>
+  </li>
+</ul>
+
+<section>
+  <h3>
+  <%= link_to "Add New Trip", passenger_trips_path(@passenger.id), method: :post %>
+  </h3>
+</section>
+
+<section class="passenger-trips">
+  <h3> Passenger Trips </h3>
+  <table id="driver-trips-table">
+    <tr>
```

Comment:
You have almost exactly the same code to render the table of trips on the passenger `show` page and the driver `show` page. Could you DRY this up with a partial somehow?